lib/Runtime/Library/TypedArray.h

```diff
@@ -217,6 +217,43 @@ namespace Js
     };
 
     template <typename TypeName, bool clamped = false, bool virtualAllocated = false>
+    class TypedArray;
+
+    // These are referenced in the TypedArray implementation, so we need to forward-typedef these.
+
+#if defined(__clang__)
+// hack for clang message: "...add an explicit instantiation declaration to .."
+#define __EXPLICIT_INSTANTINATE_TA(x) x;
+#else // MSVC
+#define __EXPLICIT_INSTANTINATE_TA(x) x
+#endif
+
```

Comment:
Looks like this block isn't doing anything very useful, except perhaps adding an extra semicolon for Clang

Reply:
Copy-paste issue from the previous version of the code.